Release_Guideline.md

```diff
@@ -0,0 +1,27 @@
+This document captures guidelines to follow when considering a new release of the BIDS specification.
+
+BIDS has generally followed a semantic versioning (semver) approach.
+The canonical semver text can be found at [https://semver.org/](https://semver.org/), and an adaptation to documents can be found at [https://semverdoc.org/](https://semver.org/).
+A specification falls somewhere between these two, so we should consider carefully the rules.
+
+In both cases, edition numbers are specified as MAJOR.MINOR.PATCH, and the difference is in the rules for incrementing each edition number.
+
+In BIDS, we have considered a major release version (the next being 2.0.0) to indicate backwards incompatible changes, and minor and patch releases must be backwards compatible.
+
+Once a decision for a release has been established, the rules of [decision-making](DECISION-MAKING) govern the mechanism of doing the release (i.e. waiting 5 business days to merge).
+
+**When it is time for a minor (1.X) release?**
+
+This release should be reserved for when a modality (BEP) has been merged into the specification.
+Special cases can override this.
+
+**When it is time for a patch (1.X.X) release?**
+
+This release can occur more frequently and in a number of cases:
+
+- A modality field has changed with the validator reflected in this change.
+- Additions or updates to the resources around BIDS (i.e. BEP update)
+- Specification rendering has been enhanced
+- Use case motivated changes
```

Comment:
```suggestion
```

Reply:
curious why a use case motived change not be a patch release rationale? 

I am thinking of a case where the change can be useful to the specification and good to get it into the stable release

Reply:
Line 20 in this PR reads: 

> This release can occur more frequently and in a number of cases, not limited to the following examples:

thus, the "use case motivated changes" are already taken care of

Reply:
I think identifying that case is useful and do not see a disadvantage of adding this particular case. It gives an idea of the variance that can capture a patch release

Reply:
> I think identifying that case is useful 

Do you think of `Use case motivated changes` as a case? I understood it as a summary for `any other cases that we cannot exhaustively specify here because we don't know what will happen` --> thus relaxing the strictness of WHEN a patch release can occur.

> do not see a disadvantage of adding this particular case

I see it as a repetition to the end of the introductory sentence ` ... , not limited to the following examples:`

> It gives an idea of the variance that can capture a patch release

I think to give an idea about the variance, the cases would have to be concrete: like in the examples before "use case motivated changes" ... whereas `use case motivated changes` is more or less saying: "any case"

Reply:
Maybe it would help to just give an example of a use case motivated change. If I recall correctly, in @effigies suggestion he pointed out that sometimes use case changes might be time sensitive. Maybe an example would be a new data sharing initiative would like to release BIDS-compliant data, and they'd like to include additional metadata than what's already available in the spec.

Reply:
Yes, the basic situation is that some non-BEP-scale changes are proposed because somebody has a dataset that is *almost* BIDS but needs a clarification or some small addition. If we release, then they can say that their dataset conforms to BIDS version 1.x.y, so it's nice not to delay too much.

To steal some of your language...

```suggestion
- A metadata field or file type is added at the request of a curator attempting to release BIDS-compliant data
```